rosidl_generator_c/CMakeLists.txt

```diff
@@ -13,8 +13,10 @@ endif()
 
 find_package(ament_cmake_python REQUIRED)
 find_package(ament_cmake_ros REQUIRED)
+find_package(rcutils REQUIRED)
```

Comment:
I don't think this needs to be found here. Instead I would `find_package()` it inside of `rosidl_generator_c_generate_interfaces.cmake` since that file creates the target that needs `rcutils` to build.

Reply:
Addressed